administrator/language/en-GB/com_cookiemanager.ini

```diff
@@ -0,0 +1,41 @@
+; Joomla! Project
+; (C) 2021 Open Source Matters, Inc. <https://www.joomla.org>
+; License GNU General Public License version 2 or later; see LICENSE.txt
+; Note : All ini files need to be saved as UTF-8
+
+COM_COOKIEMANAGER="Cookie Manager"
```

Comment:
Please alphaorder the language keys

Reply:
from my side it's done, but please check it's fine or need to edit something